docker/php/conf.d/app.ini

```diff
@@ -1,5 +1,5 @@
 upload_max_filesize = 200M ; limit of any single file uploaded via post - set high to allow big game save uploads
-memory_limit = 512M ; maximum amount of memory a script is allowed to allocate - set high for processing big game saves
+memory_limit = -1 ; maximum amount of memory a script is allowed to allocate - set high for processing big game saves
```

Comment:
we should set CLI to infinite memory, but all php request through the web server should be limited , and 512m should be more than enough for that. Let me check how to set those differently using docker